src/infra/alarm/template.yaml

```diff
@@ -148,9 +160,247 @@ Resources:
       Namespace: AWS/SQS
       Period: 60
       Statistic: Maximum
+
+  AccountDeletionProcessorQueueProcessingSlowly:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: !Sub "Account Deletion Processor Queue processing slowly. ${RunBookURL}"
+      ComparisonOperator: GreaterThanThreshold
+      Threshold: 60
+      EvaluationPeriods: 1
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: QueueName
+          Value: !Ref AccountDeletionProcessorQueueName
+      MetricName: ApproximateAgeOfOldestMessage
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Maximum
+
+  AccountDeletionProcessorQueueProcessingTooSlow:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: !Sub "Account Deletion Processor Queue processing too slow. ${RunBookURL}"
+      ComparisonOperator: GreaterThanThreshold
+      Threshold: 600
+      EvaluationPeriods: 1
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: QueueName
+          Value: !Ref AccountDeletionProcessorDLQName
+      MetricName: ApproximateAgeOfOldestMessage
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Maximum
+
+  AccountDeletionProcessorDLQProcessingSlowly:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: !Sub "Account Deletion Processor DLQ processing slowly. ${RunBookURL}"
+      ComparisonOperator: GreaterThanThreshold
+      Threshold: 60
+      EvaluationPeriods: 1
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: QueueName
+          Value: !Ref AccountDeletionProcessorDLQName
+      MetricName: ApproximateAgeOfOldestMessage
+      Namespace: AWS/SQS
+      Period: 60
+      Statistic: Maximum
+
+  AccountDeletionProcessorDLQProcessingTooSlow:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: !Sub "Account Deletion Processor Queue DLQ processing too slow. ${RunBookURL}"
+      ComparisonOperator: GreaterThanThreshold
+      Threshold: 600
+      EvaluationPeriods: 1
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: QueueName
+          Value: !Ref AccountDeletionProcessorDLQName
+      MetricName: ApproximateAgeOfOldestMessage
```

Comment:
I'm not sure if ApproximateAgeOfOldestMessage is as useful as the number of messages we have in our DLQ. @chiarapascucci-gds if you have any thoughts. This would also go for AccountDeletionProcessorQueueProcessingTooSlow.